packages/cms-lib/ignoreRules.js

```diff
@@ -3,7 +3,10 @@ const path = require('path');
 const ignore = require('ignore');
 const findup = require('findup-sync');
 
-const ignoreRules = ignore().add('node_modules');
+const ignoreRules = ignore().add([
+  'node_modules', // dependencies
+  '.*', // hidden files/folders
+]);
```

Comment:
Perhaps we should also use the [`junk`](https://www.npmjs.com/package/junk) package or something similar.
It blacklists some other things we don't care about https://github.com/sindresorhus/junk/blob/master/index.js#L3

Reply:
That is a useful package -- it looks like it only works with regex though. That may make it tricky to use with `ignore` because regex is not supported. :(

Reply:
I think we can use it as a source of inspiration and add rules to our own system.